cmd/soroban-rpc/internal/config/positive_uint32.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import (
+	"fmt"
+	"math"
+)
+
+type PositiveUint32 struct {
```

Comment:
You could simplify that so that the caller wouldn't need the `.Value` indirection. see https://go.dev/play/p/SaaFa5Zfa3h